App/Controllers/Ajax/AjaxSlugify.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Controllers\Ajax;
+
+use Core\AjaxController;
+use Cocur\Slugify\Slugify;
+use Core\JsonException;
+use Core\Traits\StringFunctions;
+
+class AjaxSlugify extends AjaxController
+{
+    use StringFunctions;
+
+    /**
+     * @return string slugified string
+     * @throws JsonException
+     */
+    public function slugifyString()
+    {
+        //only admins can update the slug
+        $this->onlyAdmin();
+        $this->onlyPost();
+        $slug = $this->request->getData("slugText-update");
+        $result = array();
+        $slugify = new Slugify();
```

Comment:
Inject in your constructor

Reply:
Resolved in branch Security_and_bug_fixes